### PR TITLE
ZCS-787 Universal UI: Compose fit and finish

### PR DESCRIPTION
**Pull request by tarangkhandelwal**
Changes for adapting to UX review suggestions.

Changeset:
1. zm.css : Reverting change from last commit and resetting line-height style value & removing height rule.
2. skin.properties: Updating styles as per review comments.

### CODE REVIEW

File: WebRoot/skins/_base/base4/skin.properties [hunk 1]
```diff
@@ -1180,7 +1180,7 @@ InviteStatusCounterBgColor  = #fff4d5
 InviteStatusCounter         = background-color: @InviteStatusCounterBgColor@; margin: @MailViewGutterSpace@ 0;
 InvCounter                  = padding:0 24px;
 
-ComposeViewContainer        = @PageBg@ @FullWidth@ @Conv2Bg@; box-sizing: border-box; padding: @MailViewGutterSpace@; padding-top: 0;
+ComposeViewContainer        = @PageBg@ @FullWidth@ @Conv2Bg@; box-sizing: border-box; padding: @MailViewGutterSpace@; padding-top: 0; border-top: 1px solid @PanelBorderColor@;
```
Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.properties]: @PanelBorder@

File: WebRoot/skins/_base/base4/skin.properties [hunk 2]
```diff
@@ -1104,13 +1104,13 @@ SearchResultColor           = #FFFEC4
 ###################
 
 AutoCompleteWaitingContainer = width: 300px; height: 70px; padding: 1.5em; box-sizing: border-box; @Text-hint@ @roundCorners(2px)@; box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5); background-color:@ListItemBgColor-normal@;
-AutoCompleteListViewContainer= @MenuContainer@ background-color:@ListItemBgColor-normal@; width:auto; padding:0px !important; border-width:1px; width: 300px; box-sizing: border-box; 
+AutoCompleteListViewContainer= @MenuContainer@ background-color:@ListItemBgColor-normal@; width:auto; padding:8px 0px; border-width:1px; width: 300px; box-sizing: border-box;
```
Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.properties]: Good that we got rid of !important :)

File: WebRoot/skins/_base/base4/skin.properties (hunk 1 above)
Comment (kalpeshhpatel) [inline WebRoot/skins/_base/base4/skin.properties]: @PanelBorder@ is for border from all side, and here only border for top is styled. So not needed, I think.

Thanks

Comment (silentsakky) [inline WebRoot/skins/_base/base4/skin.properties]: oops sorry overlooked it

**Pull request merged**